pom.xml

```diff
@@ -6,9 +6,30 @@
 	<name>Flipkart Web Automation</name>
 
 	<properties>
-		<maven.compiler.source>1.8</maven.compiler.source>
-		<maven.compiler.target>1.8</maven.compiler.target>
+		<java.version>1.8</java.version>
+		<slf4j.version>2.0.0-alpha5</slf4j.version>
```

Comment:
why do we need `sl4j` here, is it because of latest Selenium version or something else?